lib/WorseReferenceFinder/Tests/Unit/WorseReflectionTypeLocatorTest.php

```diff
@@ -104,6 +104,66 @@ public function bar()
         self::assertEquals(9, $location->offset()->toInt());
     }
 
+    public function testLocatesFirstUnion(): void
```

Comment:
this is a "temporary" solution, we can prompt the user to indicate which type to jump to with a union